### ISSUE TITLE
Ignoring fluent filtering for specific routes

### ISSUE THREAD

**Issue by knirb** [tractorcow-farm/silverstripe-fluent#709]
I'm using the FluentIsolatedExtension with DataObjects to create a view in the admin-panel that allows you to only see the DataObjects that are relevant to your locale. I would however like to be able to access all objects through graphql/any other route. Is there a way to tell fluent to ignore the filtering for some route somehow?

**Comment by tractorcow** [comment]
Yes that's a good idea; I had previously added filtering ignoring for specific admin routes, but I really should have an explicit "disabled" flag. Maybe one that you can flag on/off with a middleware?

**Comment by tractorcow** [comment]
https://github.com/tractorcow-farm/silverstripe-fluent/commit/f5c8dcef5da29711d5766f68ea3a58897d7d1b75

Let's just add another flag here, (static var with a getter / setter instead of config I think).
